src/smt/smt_engine.cpp

```diff
@@ -4431,7 +4431,7 @@ void SmtEngine::checkProof()
   if (!(
           // Pure logics
           logicString == "QF_UF" || logicString == "QF_AX"
-          || logicString == "QF_BV" ||
+          || logicString == "QF_BV" || logicString == "QF_LRA" ||
           // Non-pure logics
```

Comment:
Any reason to not include `QF_UFLRA` and other combinations?

Reply:
Good point, I've enabled it now and fixed some issues with comparing `LogicInfo`s in the process.